wp-content/themes/the-world/acf-json/group_65cdf8ec2d27d.json

```diff
@@ -0,0 +1,50 @@
+{
+    "key": "group_65cdf8ec2d27d",
+    "title": "Image Credit",
```

Comment:
Can we move this new field to the existing __Image Fields__ group? Will also want to make sure the __Image Fields__ group is shown in graphql API.

Reply:
I made some organization in the ACF groups and moved all of them to the acf theme's folder to avoid different places with these kind of configuration